ksqldb-engine/src/main/java/io/confluent/ksql/planner/plan/FlatMapNode.java

```diff
@@ -98,8 +98,9 @@ public Expression resolveSelect(final int idx, final Expression expression) {
     final ImmutableMap.Builder<Integer, Expression> buildContext = ImmutableMap
         .builder();
 
-    for (int idx = 0; idx < analysis.getSelectItems().size(); idx++) {
-      final SelectItem selectItem = analysis.getSelectItems().get(idx);
+    final List<SelectItem> selectItems = analysis.getSelectItems();
+    for (int idx = 0; idx < selectItems.size(); idx++) {
+      final SelectItem selectItem = selectItems.get(idx);
```

Comment:
Minor refactor: the `getSelectItems` walks through each item and creates another SelectItem object everytime is called. Here I just calling it once.